components/raftstore/src/store/async_io/write.rs

```diff
@@ -373,6 +375,105 @@ impl<W: WriteBatch, L: RaftLogBatch> ExtraBatchWrite<W, L> {
     }
 }
 
+/// WriteTaskBatchRecorder is a sliding window, used to record the batch size
+/// and calculate the yield interval.
+/// If the batch size is smaller than the threshold, it will return a
+/// recommended yield duration for the caller as a hint to wait for more writes.
+/// The yield interval is calculated based on the trend of the change of the
+/// batch size. The range of the trend is [0.5, 2.0]. If the batch size is
```

Comment:
Can you explain what does 2.0 mean and what does 0.5 mean? What is the quantitive relationship with wait time?

Reply:
Just a heuristic ratio according to testing records, reflecting that the time of `yield` ranges from [`yield_time` * 0.5,  `yield_time` * 2.0] nanoseconds. By default, the yield_time is `50 nanoseconds`. So, the ranges of the whole yielding is `[25 nanoseconds, 100 nanoseconds]`.